doc/release/release-notes-18.12.md

```diff
@@ -4,110 +4,103 @@
-<!-- New Samples, if applicable -->
-<!-- Other Notable Changes, if applicable -->
-
-### Zephyr Version
-
-* manifest: switch to zephyr v4.3.0-pre
-  * as the Zephyr v4.3.0 feature freeze approaches (Oct 20th), we have updated to a more recent
-    revision
-  * sending upstream pull requests to ensure we minimize patches we need to carry in-tree
-
-### Documentatation
-
-* doc: have zversion use the project settings from conf.py
-* lib: bh_arc: better documentation
-  * add doxygen for several host to smc message types
-* doxygen: Add supported boards link
-* doc: Add contribution and coding guidelines
-* docs: add visualization of CI job status
-  * graph of CI history now available on the project's landing page
-* doc: add snippet on pulling new code from main
-  * `west patch clean; west update; west patch apply`
-* lib: bh_arc: document unused commands
-
-<!-- New Boards, if applicable -->
-
-### Removed Boards
-
-* Support for `p100` has been removed
-  * `p100` (aka Scrappy) was an internal engineering unit used during initial bringup and during testing
-  * Removed support to focus on consumer-facing devices
+This release brings significant enhancements to power management, SMBus communication robustness, and developer tooling, along with improved stability across the platform.
+
+## Major Highlights
+
+- **Enhanced Power Management**: New MRISC PHY powerdown/wakeup support and power command capabilities
+- **Improved SMBus Reliability**: Enhanced error detection, logging, and robustness for CM2DM operations
+- **Advanced Developer Tooling**: New stack dump utilities and improved console tooling
+- **Zephyr v4.3.0 Alignment**: Updated to latest Zephyr pre-release for better upstream compatibility
+- **Streamlined Board Support**: Focused support on production devices
+
+## New Features
+
+### Power Management
+- **lib: bh_arc**: Added support for MRISC PHY powerdown/wakeup operations
+- **lib: bh_arc**: Implemented new Power command functionality
+- **boards: tt_blackhole**: Enabled AICLK PPM in firmware table for Galaxy boards
+
+### Communication & Protocols
+- **app: dmc**: Added support for SMBus block process call (pcall) operations
+- **drivers: smbus: stm32**: Enhanced with block read capabilities
+- **app: smc**: Added overlay configuration to build DMFW with I2C shell enabled
+
+### Developer & Debug Tools
+- **scripts**: New `dump_smc_stack.py` utility for on-demand SMC state analysis
+- **scripts: tooling**: Enhanced `tt-console` with PCIe rescan skip option for smooth TT-KMD transitions (2.3.0 to 2.4.1)
+- **scripts**: Updated `blackhole_recovery` with improved status messaging and increased delays
+
+## Stability & Robustness Improvements
+
+### Communication Reliability
+- **CM2DM Protocol**: Enhanced robustness with detection of duplicate SMBus messages and improved error handling
+- **drivers: smbus: stm32**: Added rate-limited logging for PEC (Packet Error Check) errors
```

Comment:
this has to be removed. we should eventually use the statistics feature (assuming it's still supported).
https://github.com/zephyrproject-rtos/zephyr/blob/4ef1163e39575c998af5ea00515d6efc2525a337/include/zephyr/drivers/smbus.h#L406